airbyte-connector-builder/build.gradle

```diff
@@ -10,16 +10,32 @@ airbytePython {
     moduleDirectory 'connector_builder'
 }
 
+task generateServer(type: GenerateTask){
```

Comment:
We may want to be more verbose about the name of this command to make it really explicit what it's doing. generate is a bit of an overloaded term, so maybe `generateServerFromSpec` or `FromOpenApiSpec`

Reply:
done